mongo/datadog_checks/mongo/collectors/replication_info.py

```diff
@@ -0,0 +1,51 @@
+import pymongo
+
+from datadog_checks.base.utils.common import round_value
+from datadog_checks.mongo.collectors.base import MongoCollector
+
+
+class ReplicationInfoCollector(MongoCollector):
+    """Additional replication metrics regarding the operation log. Useful to check how backed up is a secondary
+    compared to the primary."""
+
+    def __init__(self, check, tags):
+        super(ReplicationInfoCollector, self).__init__(check, "local", tags)
+
+    def collect(self, client):
+        # Fetch information analogous to Mongo's db.getReplicationInfo()
+        localdb = client[self.db_name]
+
+        oplog_data = {}
+
+        for ol_collection_name in ("oplog.rs", "oplog.$main"):
```

Comment:
What does the `ol_` prefix stand for in this method? `oplog` I think? It's not very easy to read. Maybe we can just get away with `collection_name`, `options`, etc?

Reply:
Copy/paste from before, I assume it means oplog indeed. Renaming to 'collection_name'